### DISCUSSION TITLE
[SOLVED] Does Tiny Tiny RSS account type has feed icons?

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [martinrotter/rssguard#497] [Q&A]
Migrating from other feed reader backend to tt-rss and to my surprise there are no icons, except for the default one, next to the feed title.

![](https://user-images.githubusercontent.com/10962709/133660313-f635f297-222f-474c-92db-ee958126c051.png)

 I'm using RSS Guard for several days without issues and it syncs way faster than the previous reader backend.

Edit: nothing related in the logs.

**Comment by martinrotter** [comment]
Let me check, I believe there should be icons.

**Comment by martinrotter** [comment]
OK, issue is fixed. Check with latest dev. build in 20 minutes.

**Comment by StoyanDimitrov** [comment]
I still can't see icons. New installation with removed `data4` folder, no web engine.

Edit: It still could be something on my end. The version of tt-rss is

> Tiny Tiny RSS vDebian 21~git20210204.b4cbc79+dfsg-1 (Unsupported) © 2005-2021

Don't know how recent is this.

**Reply by martinrotter** [reply]
Double tested here with  Tiny Tiny RSS v21.09-949e2ab4d

Provide debug log

https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#reprt

**Reply by StoyanDimitrov** [reply]
[debug log](https://github.com/martinrotter/rssguard/files/7224170/rss-guard.zip)

I could open the icons (https://{local-ip}/tt-rss-app//feed-icons/83.ico?1632331991) in the browser.

**Reply by martinrotter** [reply]
I see. Debug log was not of much help, it seems all okay according to it. What about trying to solve HTTPS problems - probably self-signed certificate or something, but I doubt it is the cause of your problems.

**Reply by martinrotter** [reply]
Anyway, I pushed 4fde99931 and you can test with newest dev. build and generate debug log, we will see the actual error if any in the debug log.

**Reply by StoyanDimitrov** [reply]
Currently I'm away but mentioning HTTPS I can guess what the underlying issue is. The machine is Win7 and missing TLS 1,2 and 1.3 (or whatever the current versions are). I'm pretty sure that's the problem.

I'll have a look in the following days.

**Reply by StoyanDimitrov** [reply]
I'm unable to find and download the build artifacts.

**Reply by sakkamade** [reply]
https://github.com/martinrotter/rssguard/releases/tag/devbuild

**Reply by StoyanDimitrov** [reply]
I knew I'm missing the obvious.
[debug log](https://github.com/martinrotter/rssguard/files/7242975/rssguard.zip)

Edit: It indeed needs authentication but the `Test setup` button returns green check when adding the tt-rss address to the application. I'm filling in both authentications:
> Authentication
> Requires HTTP authentication

Edit2: The icon address (https://{local-ip}/tt-rss-app//feed-icons/79.ico?1632315147) opened in the browser pops up Basic Auth dialog. As it seems the last time I test it I was logged in too.

**Reply by StoyanDimitrov** [reply]
@martinrotter Do you want me to file an issue?
